controllers/teams.js

```diff
@@ -1052,6 +1052,51 @@ router.get('/:teamId/members', async (req, res, next) => {
 			}
 		}
 
+		let files;
+
+		try {
+			files = await api(req)
+				.get('/fileStorage', {
+					qs: {
+						owner: req.params.teamId,
+					},
+				});
+		} catch (e) {
+			files = null;
+		}
+
+		/* note: fileStorage can return arrays and error objects */
+		if (!Array.isArray(files)) {
+			if ((files || {}).code) {
+				logger.warn(files);
+			}
+			files = [];
+		}
```

Comment:
You can set let files = [] and set logger.warn(files); into catch and remove this if statment